### PR TITLE
fix!: use more precise name for CLI source option

### LINKED ISSUE #179: Fix up CLI option names
We should make a separate issue to rename this to sources

_Originally posted by @korikuzma in https://github.com/cancervariants/gene-normalization/pull/169#discussion_r1146151349_

### PR DESCRIPTION
**Pull request by jsstevenson**
close #179

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,15 +73,15 @@ Use the `gene_update` command in a shell to update the database.
 
 The normalizer currently pulls data from [HGNC](https://www.genenames.org/), [Ensembl](https://useast.ensembl.org/index.html), and [NCBI](https://www.ncbi.nlm.nih.gov/gene/).
 
-To update one source, simply set `--normalizer` to the source you wish to update. The normalizer will check to see if local source data is up-to-date, acquire the most recent data if not, and use it to populate the database.
+To update one source, simply set `--source` to the source you wish to update. The normalizer will check to see if local source data is up-to-date, acquire the most recent data if not, and use it to populate the database.
```
Comment (korikuzma) [inline README.md:76]: so excited for this one

**Review by korikuzma — APPROVED** [review]

File: docs/source/managing_data/loading_and_updating_data.rst [hunk 2]
```diff
@@ -14,14 +14,15 @@ Calling the Gene Normalizer update command with the ``--update_all`` and ``--upd
 Reload individual source
 ------------------------
 
-To update specific sources, use the ``--normalizer`` option along with source name(s), quoted and separated by spaces. While it is possible to update individual source data without updating the normalized record data with ``--update_merged``, the normalization query endpoints may not function properly until normalized data is refreshed again. ::
+To update specific sources, use the ``--source`` option along with source name(s), quoted and separated by spaces. While it is possible to update individual source data without updating the normalized record data with ``--update_merged``, the normalization query endpoints may not function properly until normalized data is refreshed again. ::
+
+    gene_norm_update --source="HGNC NCBI" --update_merged
```
[the remark this answers is not in the document]
Comment (jsstevenson) [reply docs/source/managing_data/loading_and_updating_data.rst:19]: Yeah I think plural makes sense

**Review by korikuzma — DISMISSED** [review]

File: gene/cli.py [hunk 3]
```diff
@@ -229,11 +229,11 @@ def _load_merge(db: AbstractDatabase, processed_ids: Set[str]) -> None:
     is_flag=True,
     help='Update concepts for normalize endpoint from accepted sources.'
 )
-def update_normalizer_db(normalizer: str, aws_instance: bool, db_url: str,
+def update_normalizer_db(sources: str, aws_instance: bool, db_url: str,
                          update_all: bool, update_merged: bool) -> None:
     """Update selected normalizer source(s) in the gene database.
 
-    :param normalizer: names of sources to update, comma-separated
+    :param source: names of sources to update, comma-separated
```
Comment (korikuzma) [inline gene/cli.py:236]:
```suggestion
    :param sources: names of sources to update, comma-separated
```

**Review by korikuzma — CHANGES_REQUESTED** [review]

File: gene/cli.py [hunk 4]
```diff
@@ -102,18 +102,18 @@ def dump_database(output_directory: Path, db_url: str):
...
 
-def _update_normalizers(
-    normalizers: Collection[SourceName], db: AbstractDatabase, update_merged: bool
+def _update_normalizer(
+    sources: Collection[SourceName], db: AbstractDatabase, update_merged: bool
 ) -> None:
     """Update selected normalizer sources.
 
-    :param normalizers: names of sources to update
+    :param source: names of sources to update
```
Comment (korikuzma) [inline gene/cli.py:110]: sorry i missed this one
```suggestion
    :param sources: names of sources to update
```

**Review by korikuzma — APPROVED** [review]

**Pull request merged**